crates/re_renderer/shader/utils/sphere_quad.wgsl

```diff
@@ -27,9 +27,9 @@ fn sphere_quad_span_perspective(
     let camera_offset = radius_sq * distance_to_camera_inv;
     var modified_radius = point_radius * distance_to_camera_inv * sqrt(distance_to_camera_sq - radius_sq);
 
-    // We're computing a coverage mask in the fragment shader - make sure the quad doesn't cut off our antialiasing.
+    // Add half a pixel of margin for the feathering we do for antialiasing the spheres.
     // It's fairly subtle but if we don't do this our spheres look slightly squarish
-    modified_radius += frame.pixel_world_size_from_camera_distance * camera_distance;
+    modified_radius += 0.5 * frame.pixel_world_size_from_camera_distance * camera_distance;
```

Comment:
thanks for following up on this. Given the previous change this should be safe, but have you checked if the point cloud in the multiview demo looks the exact same before after? (you can stop the movement with space and then take zoomed in screenshots if needed)